serialization.md

```diff
@@ -75,7 +75,7 @@ This specification uses the following terms:
         A collection of tags grouped under a common prefix (the name component before <code>:</code>).
 	For example, in an image tagged with the name <code>my-app:3.1.4</code>, <code>my-app</code> is the <i>Repository</i> component of the name.
 	A repository name is made up of slash-separated name components, optionally prefixed by a DNS hostname.
-	The hostname must follow comply with standard DNS rules, but may not contain <code>_</code> characters.
+	The hostname must comply with standard DNS rules that apply to IDNA labels as defined in <a href="https://tools.ietf.org/html/rfc5890#section-2.3">RFC 5890</a>.
```

Comment:
I think we want to [explicitly list A-label, U-label, or NR-LDH label][1].  Searching the spec for a definition for “IDNA labels” turned up [this][2]:

> IDNA labels come in two flavors: an ACE-encoded form and a Unicode (native character) form.  These are referred to as A-labels and U-labels, respectively…

But I think we want to allow [NR-LDH labels][3] as well, because U-labels must contain [at least one non-ASCII character][4], and I'm fine with all-ASCII, non-A label, repository names.

[1]: https://github.com/opencontainers/image-spec/pull/163#discussion_r75527471
[2]: https://tools.ietf.org/html/rfc5890#section-2.3.1
[3]: https://tools.ietf.org/html/rfc5890#section-2.3.2.2
[4]: https://tools.ietf.org/html/rfc5890#section-2.3.2.1